installer/gentoo/prepare

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh -e
+# Copyright (c) 2017 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+DISTROAKA='gentoo'
+
+# Set global use flags, separated by a space
+set_flags() {
+    while [ "$#" != 0 ]; do
+         local flag=`euse -a $1 | cut -d\[ -f1`;
```

Comment:
What's the output you are parsing here?  There might be a cleaner way to do this.